src/core/qgsinterval.h

```diff
@@ -43,13 +44,29 @@ class CORE_EXPORT QgsInterval
     //! Seconds per year (average)
     static const int YEARS = 31557600;
     //! Seconds per month, based on 30 day month
+#ifndef SIP_RUN
     static const int MONTHS = 60 * 60 * 24 * 30;
```

Comment:
I just modified the sipify script to remove the static const value assignment, so you should be able to rebase and remove the #ifdefs